packages/gatsby-image/src/__tests__/__snapshots__/index.js.snap

```diff
@@ -29,13 +29,13 @@ exports[`<Img /> should render fixed size images 1`] = `
         alt="Alt text for the image"
         height="100"
         src="test_image.jpg"
-        style="position: absolute; top: 0px; left: 0px; width: 100%; height: 100%; opacity: 1;"
+        style="position: absolute; top: 0px; left: 0px; width: 100%; height: 100%; font-family: "object-fit: cover; object-position: 50% 50%"; opacity: 1;"
```

Comment:
Show this weirdness (necessary for the polyfill, I know!) make its way into the unit tests? I wouldn't have thought the opt-in behavior would have impacted this.

Reply:
That's my bad! I don't know anything about testing, but saw that tests were failing, so fumbled around a bit trying to make the markup match the updated version.

I'm totally happy to drop these changes and I defer to you about how to handle the test failures below. 🤷‍♂️

Reply:
you can run `yarn jest -u` to update snapshots in your repository and commit those changes

Reply:
Thank you @pieh!